### ISSUE TITLE
Bare metal Install

### ISSUE THREAD

**Issue by john-clark** [pirate/ArchiveBox#402]
Bare metal and Publishing wiki articles did not provide results desired. https://github.com/pirate/ArchiveBox/wiki/Publishing-Your-Archive only provides read-only access. My install was on headless Debian 10, running "archive server" only provides local access, and do not have access to GUI. This is not for world-readable sites just internal testing usage. Would be better to provide a non-proxy solution but I just wanted to be able to see the program operation. 

```bash
apt install python3 python3-pip git curl nginx wget youtube-dl chromium
pip3 install archivebox
mkdir /usr/local/data
chmod 777 /usr/local/data
vi /etc/nginx/sites-enabled/default
```
```nginx
events {
  worker_connections  4096;  ## Default: 1024
}
http {
 server {
   listen 80;
   listen [::]:80;
 
   server_name _;
 
   location / {
       proxy_pass http://127.0.0.1:8000/;
       proxy_set_header Host $host;
   }
 }
}
```
```bash
service nginx restart
vi /etc/systemd/system/archivebox.service
```
```ini
[Unit]
Description=ArchiveBox service
After=network.target
StartLimitIntervalSec=0

[Service]
Type=simple
Restart=always
RestartSec=1
User=youruser
WorkingDirectory=/usr/local/data/
ExecStart=archivebox server

[Install]
WantedBy=multi-user.target
```
```bash
cd /usr/local/data
archivebox init
systemctl enable archivebox
systemctl start archivebox
```

**Comment by pirate** [comment]
Unfortunately to add sites from the UI you need to create a user to log in as, even if the archived files themselves are public. This is a fundamental limitation of the Django admin requiring user authentication, and it's not something I want to change as it's a lot of effort for relatively little gain.

The recommended solution if you want both archive read and *write* access to be public is to do the following:
```bash
cd /usr/local/data
archivebox config --set PUBLIC_INDEX=True
archivebox config --set PUBLIC_SNAPSHOTS=True
archivebox manage createsuperuser
# create it as "public" / "public" or something easy to remember
archivebox server
open http://127.0.0.1:8000
# non-logged in users will be able to view everything
# logged-in users (public/public) will be able to view, *add*, *edit*, and *manage* your archive
```

If I misunderstood your question or if this doesn't answer your question, feel free to comment back and I'll reopen the ticket (a screenshot of what you're seeing would help too).

**Comment by john-clark** [comment]
Yes, misunderstanding, no work requested, above solution is working fine. Systemd runs as the user I ran archivebox init with. This was an attempt to document the steps I took to be able to remotely get to archivebox login and web interface from remote computer.  https://github.com/pirate/ArchiveBox/wiki/Publishing-Your-Archive is a read only non-interactive page. I wanted to be able to see the django server hosted at 127.0.01:8000 on my management pc.


![image](https://user-images.githubusercontent.com/3967451/88955832-1a69f100-d262-11ea-9768-3e4dc26ca23b.png)
